solidity/contracts/Gravity.sol

```diff
@@ -521,6 +531,7 @@ contract Gravity is ReentrancyGuard, AccessControl, Pausable, Ownable {
 		ValsetArgs calldata _currentValset,
 		// These are arrays of the parts of the validators signatures
 		ValSignature[] calldata _sigs,
+		address _paymentAddress,
```

Comment:
For this `_paymentAddress` parameter input by relayer, there is no check on it. Any reasonable explanation on it?

Reply:
its the address inputted by the relayer to receive the fee. Is there any security risks?

Reply:
No, it's just normal using practice to prevent from some mistakes. If a mistaken input 0 address could cause that the fee cannot be received by anyone.

Reply:
yeah I think in that case we dont care.

we need to optimize the smart contract to consume minimum gas. otherwise you could perform validation on all those addresses (zero address, dead adddress, 0x01 etc...)